packages/Webkul/Admin/src/Config/system.php

```diff
@@ -1147,13 +1147,13 @@
                 'options'       => [
                     [
                         'title' => 'admin::app.configuration.index.sales.payment-methods.pending',
-                        'value' => 'pending',
+                        'value' => \Webkul\Sales\Models\Order::STATUS_PENDING,
```

Comment:
We mean to say like this,

~~~php
use Webkul\Sales\Models\Order;
...
'value' => Order::STATUS_PENDING
~~~